test/wolt_modal_sheet_test.dart

```diff
@@ -169,7 +169,7 @@ void main() {
     await tester.tap(find.text('Open sheet'));
     await tester.pumpAndSettle();
 
-    expect(tester.takeException(), isNotNull);
+    expect(tester.takeException() != null, true);
```

Comment:
Existing check was correct way to test this.

Reply:
Reverted back here: b3bdd338de08ec7e60431e7fcd61f8c5dcdea9ae